robocop/checkers/lengths.py

```diff
@@ -502,6 +561,60 @@ def visit_Keyword(self, node):  # noqa: N802
                 sev_threshold_value=key_calls,
             )
 
+    def _visit_keyword_arguments(self, keyword_node: Keyword, arguments: Arguments):
+        arg_node = arguments.get_token(Token.ARGUMENTS)
+
+        arg_count = len(arguments.values)
+        if arg_count > self.param("too-many-arguments", "max_args"):
+            self.report(
+                "too-many-arguments",
+                keyword_name=keyword_node.name,
+                arguments_count=arg_count,
+                max_allowed_count=self.param("too-many-arguments", "max_args"),
+                node=arg_node,
+                lineno=arg_node.lineno,
+                col=arg_node.col_offset + 1,
+                end_col=arg_node.col_offset + len(arg_node.value) + 1,
+                extended_disablers=(keyword_node.lineno, keyword_node.end_lineno),
+                sev_threshold_value=arg_count,
+            )
+
+        required_arg_count = 0
+        optional_arg_count = 0
+        for arg in arguments.values:
+            if "}=" in arg:
```

Comment:
We may want to ignore errors as well, for example:

```
[Arguments]    ${argument} = default
```

Not sure how it behaves but most likely it's parsing error, but may still come up here as 'optional' argument